### ISSUE TITLE
Feature Request - Straightforward way to access 24x7 recordings

### ISSUE THREAD

**Issue by mcondren** [blakeblackshear/frigate#839]
I understand the goal with Frigate is probably not to evolve it into an NVR, however with the new 24x7 recording feature I am struggling to find a way to make these 60sec recordings useful. A full-on timeline scrubber may not be on-path but some way to be able to easily start a stitched together stream from a specific camera (or alternately all of them in a grid) starting at a specific date and time would allow the  review of saved footage that doesn’t necessarily have a valid detection, and therefore no event. I normally keep a weeks worth of raw footage in the event I need to review footage later that may not have risen to the level of a detection or contain objects I wasn’t tracking. Eg. after a burglary that didn’t set off a detection , for whatever reason.

**Comment by blakeblackshear** [comment]
The entire goal was to create something that didn't require manually reviewing motion events on a timeline, so I'm not really interested in spending a bunch of time (d)evolving frigate into just another NVR with timeline scrubbing. I would much rather continue to invest in improving the parts that make it even less likely to need to review 24/7 recordings. I have paired frigate with 2 weeks worth of 60 second segments as a last resort for about 3 years at two homes, and I can count on 1 hand the number of times I have needed to go back and look at older footage. In nearly every case, I have known what time I was interested in, so finding the clip based on the minute hasn't really been a problem. I realize my circumstances are probably quite a bit different than others.

I _am_ interested in making it easier to use the 24/7 recordings to improve real time detection. I suspect that some of the features I have in mind surrounding that will make it easier to review historical footage as a byproduct, but it won't be the primary use case.
